### ISSUE TITLE
bulk_update_* doesn't do anything

### ISSUE THREAD

**Issue by trickvi** [ckan/ckan#2662]
Running: 

```python
>>> r = requests.post('http://localhost:5000/api/3/action/bulk_update_private', json={'datasets':['a-dataset'], 'org_id': 'another-organization'}, headers={'Authorization':apikey})
>>> r.json()
{u'help': u'http://localhost:5000/api/3/action/help_show?name=bulk_update_private', u'success': True, u'result': None}
```

Seems to work, but doesn't. The dataset is still public.

**Comment by trickvi** [comment]
This also applies to ``bulk_update_public`` and ``bulk_update_delete``. Changing title accordingly.

**Comment by amercader** [comment]
Fixed in #3199
